pkg/cmdline/cmdline.go

```diff
@@ -36,8 +43,26 @@ func AddFlags(cmd *cobra.Command, opts *Options) {
 	cmd.Flags().UintVarP(&opts.MemoryMiB, "memory", "m", 512, "virtual machine RAM size in mibibytes")
 
 	cmd.Flags().StringVarP(&opts.TimeSync, "timesync", "t", "", "sync guest time when host wakes up from sleep")
-
 	cmd.Flags().StringArrayVarP(&opts.Devices, "device", "d", []string{}, "devices")
 
 	cmd.Flags().StringVar(&opts.LogLevel, "log-level", "", "set log level")
+	cmd.Flags().StringVar(&opts.RestfulURI, "restful-uri", DefaultRestfulURI, "URI address for RestFul services")
```

Comment:
Any reason for using `none://` instead of `""` as the default?

Reply:
i prefer this approach because the default IS none and this saves us a conditional check later ...